.gitignore

```diff
@@ -165,6 +164,7 @@ perfdata.db
 
 .idea/
 _dotgraphs/
```

Comment:
Do you wanna remove this?

Reply:
backwards compatibility with existing git repos :P